lib/plugins/entities.js

```diff
@@ -136,8 +138,10 @@ function inject (bot, { version }) {
     // spawn object/vehicle
     const entity = fetchEntity(packet.entityId)
     let entityData = objects[packet.type]
-    if (entityData === undefined && bot.majorVersion !== '1.13') {
-      entityData = entitiesArray.find(entity => entity.internalId === packet.type)
+    if (entityData === undefined) {
+      if (bot.majorVersion === '1.8' || bot.majorVersion === '1.9' || bot.majorVersion === '1.10' || bot.majorVersion === '1.11' || bot.majorVersion === '1.12') {
+        entityData = entitiesArray.find(entity => entity.internalId === packet.type)
+      }
```

Comment:
The packet.type sent by spawn_entity and spawn_entity_living are different maps in 1.13. And mcData does not have a spawn_entity map.

Example
boat | 1
item | 2
area_effect_cloud | 3
chest_minecart | 10
command_block_minecart | 10
furnace_minecart | 10
hopper_minecart | 10
minecart | 10
spawner_minecart | 10
tnt_minecart | 10
tnt | 50
end_crystal | 51
arrow | 60
snowball | 61
egg | 62
fireball | 63
small_fireball | 64
ender_pearl | 65
wither_skull | 66
shulker_bullet | 67
llama_spit | 68
falling_block | 70
item_frame | 71
eye_of_ender | 72
potion | 73
experience_bottle | 75
firework_rocket | 76
leash_knot | 77
armor_stand | 78
evoker_fangs | 79
fishrod | 90
spectral_arrow | 91
trident | 94